test/decimal128_test.dart

```diff
@@ -0,0 +1,305 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+import 'dart:math';
+
+import 'package:meta/meta.dart';
+import '../lib/src/native/realm_core.dart';
+import 'package:test/test.dart';
+
+const int defaultTimes = 100000;
+
+void repeat(dynamic Function() body, [int times = defaultTimes]) {
+  for (var i = 0; i < times; ++i) {
+    body();
+  }
+}
+
+@isTest
+void repeatTest(String description, dynamic Function(Decimal128 x, int xInt, Decimal128 y, int yInt) body, [int times = defaultTimes]) {
```

Comment:
I can't invoke single test from command line or vs code. Could we make this work

Reply:
I mean I tried that in VS code. Is it working for you?

Reply:
I just tried and I get the same error 
> No tests match regular expression "^Decimal128 divide( \(variant: .*\))?$".
and I think I have the latest versions of the VSCode extensions

Reply:
Yes. How do you invoke it?

Reply:
Also if I do 
dart test -n "device" -r expanded in the repo the `decimal` tests are not executed

Reply:
Fixed